Mathlib/GroupTheory/GroupAction/Group.lean

```diff
@@ -178,6 +180,36 @@ theorem smul_eq_iff_eq_inv_smul (g : α) {x y : β} : g • x = y ↔ x = g⁻¹
 
 end Group
 
+section Monoid
+
+variable [Monoid α] [MulAction α β]
+
+@[to_additive]
+instance : MulAction αˣ β where
```

Comment:
I'm pretty sure we already have this instance in another file? It's _maybe_ ok to move it here, but we shouldn't have two copies of it

Reply:
Yes, with the imports I subsequently added, that now appears to be available, so I have removed this instance.